src/test/java/games/strategy/test/EqualsPredicateTests.java

```diff
@@ -159,7 +164,8 @@ public void shouldReturnFalseWhenOwnerObjectsAreNotEqual() {
       assertFalse(equalsPredicate.test(owner1, owner2));
     }
 
-    private static final class FakeOwnerClass {
+    @Nested
+    private final class FakeOwnerClass {
```

Comment:
This class does not contain tests, and thus does not need to be annotated with `@Nested`.